### ISSUE TITLE
I can't find the UpdateReportAcknowledgements !

### ISSUE THREAD

**Issue by iamtaojie** [amzn/selling-partner-api-models#135]
Hi all,

I can't find the UpdateReportAcknowledgements in the sp-api !

Can you tell me how to change the Report Acknowledged like the UpdateReportAcknowledgements

Best regards,
T

**Comment by heitke** [comment]
@iamtaojie What I'm hearing from Amazon is that they are deprecating this feature. They are open to revisiting it if there is a valid use-case for it, but for now we are proceeding as if they are going to deprecate the ability to acknowledge reports.

**Comment by iamtaojie** [comment]
@heitke So is there any other way to tell if I downloaded the report

**Comment by heitke** [comment]
@iamtaojie For the scheduled reports or auto-generated reports, the current plan is to basically keep track on our end.  We will periodically request a list of the latest 10 or 20 reports and compare the list of IDs to the ones we've already processed.  Once we download and process a report we will keep track of that so we don't reprocess that report.

And of course for the reports that we request we will already know if we've processed it yet or not.

It's not a perfect system - having the acknowledgement system sure was handy.  But I think we should be able to work around it.
